Markup/css/styles.css

```diff
@@ -2142,10 +2141,6 @@ svg .outer {
     .app-desc h2{
         margin-top:20px;
```

Comment:
Space after `:` is still there in other places. Please check.

Reply:
fixed with #14